server/routes/home.js

```diff
@@ -26,6 +26,39 @@ export const getRoot = async (req, res) => {
   });
 };
 
+export const getTeams = (req, res) => {
```

Comment:
How come there is a `getTeams` function in both home.js as well as in ministryteams.js? It looks like you only use the one from ministryteams.js, so this looks like dead code.